src/config.rs

```diff
@@ -67,6 +67,15 @@ impl ProjectPathsConfig {
         let graph = Graph::resolve_sources(self, sources)?;
         let ordered_deps = collect_ordered_deps(&flatten_target, self, &graph)?;
 
+        #[cfg(windows)]
+        let ordered_deps = {
+            use path_slash::PathBufExt;
+            ordered_deps
```

Comment:
`for p in &mut ordered_deps { *p = ... }`?
or since we only iterate it doesn't need collect